src/app/models/form_feedback.server.model.ts

```diff
@@ -14,6 +15,13 @@ const FormFeedbackSchema = new Schema<IFormFeedbackSchema, IFormFeedbackModel>(
       ref: FORM_SCHEMA_ID,
       required: true,
     },
+    submissionId: {
+      type: Schema.Types.ObjectId,
+      ref: SUBMISSION_SCHEMA_ID,
+      // TODO #3964: Update to true once we fully migrate to /submissions/{submissionId}/feedback endpoint
+      required: false,
+      index: true,
```

Comment:
just for consistency across the codebase, can we use the `.index` syntax for this? can look at the other model files for examples